clang/lib/Headers/ia32intrin.h

```diff
@@ -173,25 +183,59 @@ __popcntq(unsigned long long __A)
 #endif /* __x86_64__ */
 
 #ifdef __x86_64__
+/// Returns the program status and control \c RFLAGS register with the \c VM
+///    and \c RF flags cleared.
+///
+/// \headerfile <x86intrin.h>
+///
+/// This intrinsic corresponds to the \c PUSHFQ + \c POP instruction sequence.
```

Comment:
PUSHFQ->PUSHF

Reply:
The mnemonic in SDM is PUSHFQ

Reply:
I wrote that comment before I checked and meant to remove it before I approved. Sorry.